test/conformance/tests/sriov_operator.go

```diff
@@ -1079,28 +1099,7 @@ func createSriovPolicy(sriovDevice string, testNode string, numVfs int, resource
 		resNum, _ := testedNode.Status.Allocatable[corev1.ResourceName("openshift.io/"+resourceName)]
 		capacity, _ := resNum.AsInt64()
 		return capacity
-	}, 10*time.Minute, time.Second).Should(Equal(int64(numVfs)))
-}
-
-func createUnschedulableTestPod(node string, networks []string, resourceName string) {
-	podDefinition := pod.RedefineWithNodeSelector(
-		pod.DefineWithNetworks(networks),
-		node,
-	)
-	createdPod, err := clients.Pods(namespaces.Test).Create(podDefinition)
-	Consistently(func() k8sv1.PodPhase {
-		runningPod, err := clients.Pods(namespaces.Test).Get(createdPod.Name, metav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		return runningPod.Status.Phase
-	}, 3*time.Minute, 1*time.Second).Should(Equal(k8sv1.PodPending))
-	pod, err := clients.Pods(namespaces.Test).Get(createdPod.Name, metav1.GetOptions{})
-	Expect(err).ToNot(HaveOccurred())
-	for _, condition := range pod.Status.Conditions {
-		if condition.Reason == "Unschedulable" && strings.Contains(condition.Message, "Insufficient openshift.io/"+resourceName) {
-			return
-		}
-	}
-	Fail("Pod should be Unschedulable due to: Insufficient openshift.io/" + resourceName)
+	}, 15*time.Minute, time.Second).Should(Equal(int64(numVfs)))
```

Comment:
why you put a long wait here?

Reply:
Sometimes 10 minutes is not enough for Node to update capacity.